spec/steps/execute.fmf

```diff
@@ -45,6 +45,28 @@ example: |
             how: beakerlib
             isolate: true
 
+/avc_check:
+    summary: Check for SELinux messages, default: true
+    description:
+        Optional boolean attribute `avc_check` can be used to
+        disable check for SELinux messages.
+    example: |
+        execute:
+            how: beakerlib
+            avc_check: false
+
+/plugins:
+    summary: Additional plugin to be used
+    description:
+        Names of the plugins to be run during test execution
+        and optionally their configuration as key: value pairs
+    example: |
+        executute:
+            how: beakerlib
+            plugins:
+                scl_enable:
+                    enable: rh-python27 httpd24
```

Comment:
Why wouldn't you simply have it on one `scl_enable` line?

Also, this would mean the SCLs need to be installed first, right? Note that the instead of `scl_enable` scriplet I'd rather use the `scl` executable, so this could be all handled on a `python` level.

@psss WDYT?